### PR TITLE
Violin multi

### PR DESCRIPTION
**Pull request by moriondo2022**
Gene expression multiplot on Gene page.

### CODE REVIEW

File: src/views/Gene/Template.vue [hunk 1]
```diff
@@ -1,3 +1,6 @@
+
+import ResearchExpressionPlot from '../../components/researchPortal/ResearchExpressionPlot.vue';
```
Comment (qu-y) [inline src/views/Gene/Template.vue:2]: another way to link is to start with **_@_** which is equivalent to **_src_** folder, if you find yourself linking from too many levels _../.._

File: src/components/researchPortal/ResearchExpressionPlot.vue [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+                });
+            }
+            return dataRows;
+        },
+        rows(){
+            return this.tableData.length;
+        }
+    },
+	watch: {
+        rawData: function(){
```
[the remark this answers is not in the document]
Comment (moriondo2022) [reply src/components/researchPortal/ResearchExpressionPlot.vue]: Refactored all uses of `function` keyword - thank you for suggesting.

File: src/components/researchPortal/ResearchSummaryPlot.vue [hunk 3]
```diff
@@ -1,7 +1,11 @@
 <template>
 <div class="chart-wrapper">
     <div v-if="!!summaryPlot['title']"><h4>{{summaryPlot['title']}}</h4></div>
-    <div class="all-charts">
+    <research-multi-plot v-if="summaryPlot.type == 'multi'"
+				v-bind:summaryPlot="summaryPlot"
+                v-bind:rawData="rawData">
```
[the remark this answers is not in the document]
Comment (moriondo2022) [reply src/components/researchPortal/ResearchSummaryPlot.vue]: Refactored to use :rawData convention.

File: src/views/Gene/Template.vue (hunk 1 above)
Comment (moriondo2022) [reply src/views/Gene/Template.vue]: Done.

**Pull request merged**